src/View/HTML.php

```diff
@@ -82,7 +82,7 @@ public static function createTag($tag, $attributes, $content = null)
             if ($content) {
                 throw new InvalidArgumentException("Void element \"{$tag}\" cannot have content");
             }
-            return "<{$tag}{$preparedAttributes} />";
+            return "<{$tag}{$preparedAttributes}>";
```

Comment:
No self-closing tags for void elements in html5 - https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags